content/docs/quickstart.mdx

```diff
@@ -22,74 +22,216 @@ keywords:
   ]
 ---
 
-import ConfigDocker from '@site/content/examples/config/config.docker.yaml.md';
-import DockerCompose from '@site/content/examples/docker/basic.docker-compose.yml.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Pomerium Zero Quickstart
+
+The Zero Quickstart shows you how to install and run Pomerium Zero in a Docker container.
+
+## Before you start
+
+- Install [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- Create a (free) [**Pomerium Zero account**](https://console.pomerium.app/create-account)
+
+## Get Pomerium Zero configuration
+
+In the Zero onboarding screen, select the **Docker** tab and copy the Docker Compose configuration. (If you selected **Finish** before copying the Docker configuration, we've provided a copy below.)
```

Comment:
It's probably worth mentioning something here about how you get to the onboarding screen, maybe something like "after creating an account, you should see an onboarding page"?

Reply:
Good point, I'll add that in.